packages/core/lib/commands/test/copyArtifactsToTempDir.js

```diff
@@ -19,11 +18,11 @@ const copyArtifactsToTempDir = async config => {
     return {temporaryDirectory};
   }
 
-  await promisify(copy)(config.contracts_build_directory, temporaryDirectory);
+  await copy(config.contracts_build_directory, temporaryDirectory);
```

Comment:
Any reason this doesn't use `overwrite` like the other one?

Reply:
Nice catch, this will definitely need it if a temporary directory is being created first!

Reply:
I'm confused, this actually seems to work without the `overwrite: true` on `truffle test` on the metacoin box. I have no idea why!

Reply:
Ah I think it ends up picking up and compiling the artifacts later when it goes to do all the test-specific stuff like `Assert.sol`.